app/utils/steam/steamcmd/wrapper.py

```diff
@@ -139,6 +139,50 @@ def check_symlink(self, link_path: str, target_local_folder: str) -> None:
 
             CreateJunction(target_local_folder, link_path)
 
+    def create_symlink(self, runner: RunnerPanel, symlink_source_path: str, symlink_destination_path: str) -> None:
+        try:
+            runner.message(
+                f"[{symlink_source_path}] -> " + symlink_destination_path
+            )
+            if os.path.exists(symlink_destination_path):
+                logger.debug(
+                    f"Removing existing link at {symlink_destination_path} and recreating link to {symlink_source_path}"
+                )
+                # Remove by type
+                if os.path.islink(
+                    symlink_destination_path
+                ) or os.path.ismount(symlink_destination_path):
+                    os.unlink(symlink_destination_path)
+                elif os.path.isdir(symlink_destination_path):
+                    shutil.rmtree(symlink_destination_path)
+                else:
+                    os.remove(symlink_destination_path)
+            if self.system != "Windows":
+                os.symlink(
+                    symlink_source_path,
+                    symlink_destination_path,
+                    target_is_directory=True,
+                )
+            elif sys.platform == "win32":
+                from _winapi import CreateJunction
+
+                CreateJunction(
+                    symlink_source_path, symlink_destination_path
+                )
+            self.setup = True
+            runner.message(
+                "Finished creating symlink\n"
+            )
+        except Exception as e:
+            runner.message(
+                f"Failed to create symlink. Error: {type(e).__name__}: {str(e)}"
+            )
+            show_fatal_error(
```

Comment:
Don't use a fatal error for this. It should only be used for true fatal issues since it has extra stuff like log upload, etc. I'd instead just use a warning box.